docs/3-customizing/05-css-modules.md

```diff
@@ -0,0 +1,78 @@
+|---------|---------------------------------------------------------------|
+| PostCSS | PostCSS supports CSS Modules through the plugin `postcss-modules`. [Learn more](https://www.npmjs.com/package/postcss-modules) |
+| Webpack | The `css-loader` has CSS Modules built-in. You can activate it by using the `?modules` flag. [Learn more](https://github.com/webpack-contrib/css-loader) |
+| Vite    | Vite supports CSS Modules through Lightning CSS. [Learn more](https://vitejs.dev/guide/features#css-modules) |
+
+### Frameworks
+
+| Name   | Description    |
+|--------|----------------|
+| React  | Integration into the build tool might be needed. |
+| Angular| Integration into the build tool might be needed. |
+| Vue    | Integration into the build tool might be needed. |
+| Svelte | Integration into the build tool might be needed. |
+
+## Example Usage
+As mentioned earlier, each tool that supports CSS modules resolves them to a JavaScript import that has a default export as a JavaScript Object with keys for each CSS definition and named exports respectively.
+
+### Usage with tooling
+
+With the current example, we cover the use case for interactive `ui5-icon` inside `ui5-input`. By default, not every `ui5-icon` placed inside `ui5-input` should have an interactive visual appearance, and people had to implement missing styles themselves. With `@ui5/webcomponents/dist/styles/Icon.module.css`, we provide styles for interactive `ui5-icon` that might be reused.
+
+```html
+<ui5-input>
+    <ui5-icon id="custom-icon" name="search" slot="icon"></ui5-icon>
+</ui5-input>
+<script type="module">
+    import { inputIcon } from "@ui5/webcomponents/dist/styles/Icon.module.css";
+
+    const icon = document.getElementById("custom-icon");
+    icon.classList.add(inputIcon);
+</script>
+```
+
+### Usage without Tooling
+
+CSS modules can be used without being processed by build tools, similar to native CSS files. To use them, you need to import them directly inside your CSS files using a relative path to the module. Our CSS definitions follow the camelCase naming convention, meaning that each key from the JavaScript Object or named import has a class representation (for example, `import { inputIcon } from "@ui5/webcomponents/dist/styles/Icon.module.css"'` can be used as the `.inputIcon` class in this scenario).
+
+**Note:** If you are unsure whether your application or build tool supports CSS modules, you can use the respective file without the `.module` in the file name, which is provided for each of the CSS modules.
+
+**Note:** When CSS modules are imported into CSS files, the CSS definitions are not scoped.
+
+```html
+<style>
+    @import url("./node_modules/@ui5/webcomponents/dist/styles/Icon.module.css");
+    ## Or
+    @import url("./node_modules/@ui5/webcomponents/dist/styles/Icon.css");
+</style>
+<ui5-input>
+    <ui5-icon name="search" slot="icon" class="inputIcon"></ui5-icon>
+</ui5-input>
+```
+
+## Available CSS Modules in Main Package
```

Comment:
Main package -> maybe change to `@ui5/webcomponents` as some users might not know that we call it main